packages/vuetify/src/components/VColorPicker/VColorPickerPreview.ts

```diff
@@ -33,9 +33,9 @@ export default Vue.extend({
           max: 1,
         },
         style: {
-          backgroundImage: !this.disabled
-            ? `linear-gradient(to right, transparent, ${RGBtoCSS(this.color.rgba)})`
-            : undefined,
+          backgroundImage: this.disabled
+            ? undefined
+            : `linear-gradient(to ${this.$vuetify.rtl ? 'left' : 'right'}, transparent, ${RGBtoCSS(this.color.rgba)})`,
```

Comment:
maybe pull this out into a computed prop `gradient`?

Reply:
I was thinking about it but seemed to me as unnecessary premature optimization